File: level_1/PYTHON/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    arr.sort()
    
    if divisor==1:
        return arr
    
    answer = []
    
    for i in arr:
        if i%divisor==0:
            answer.append(i)

    if len(answer)==0:
        answer.append(-1)
    
    return answer

File: level_1/C++/나누어 떨어지는 숫자 배열.cpp
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr, int divisor) {
	vector<int> answer;
	if (divisor == 0) return arr;

	sort(arr.begin(), arr.end());

	for (int elem : arr) {
		if (elem%divisor == 0) {
			answer.push_back(elem);
		}
	}

	if (answer.empty() == 1) answer.push_back(-1);

	return answer;
}